### PR TITLE
fix/update pages qwik guide

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: content/pages/framework-guides/deploy-a-qwik-site.md [hunk 1]
```diff
@@ -17,15 +17,22 @@ Create a new project by running the following command in your terminal:
 $ npm create qwik@latest
 ```
 
-You will be prompted to select a starter, of which you can choose any of the available options. Then you will be prompted to choose a server; this guide uses Cloudflare Pages.
+You will need to provide a directory for your project (for the rest of this guide we'll assume you chose `./qwik-app`) and select a starter from a provided list, choose any of the QwikCity available options.
```
Comment (dario-piotrowicz) [inline content/pages/framework-guides/deploy-a-qwik-site.md]: Note: here "QwikCity" is not a typo, in the cli they call it "QwikCity" (even though the meta-framework's called "Qwik City")

I just figured I could be consistent with their inconsistency? 🤔

Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-qwik-site.md]:
```suggestion
You will need to provide a directory for your project (this guide will assume you chose `./qwik-app`) and select a starter from a provided list. Choose any of the QwikCity available options.
```

File: content/pages/framework-guides/deploy-a-qwik-site.md [hunk 2]
```diff
@@ -17,15 +17,22 @@ Create a new project by running the following command in your terminal:
...
 
-By selecting _Cloudflare Pages_ as your server option in the terminal drop-down menu, your folder will have a `functions/[[path]].ts` file. The `[[path]]` filename indicates that this file will handle requests to all incoming URLs; refer to [Path segments](/pages/platform/functions/#path-segments) to learn more.
+The command will also ask you if you would like to install npm dependencies, chose yes.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-qwik-site.md]:
```suggestion
The command will also ask you if you would like to install npm dependencies, choose yes.
```

File: content/pages/framework-guides/deploy-a-qwik-site.md [hunk 3]
```diff
@@ -17,15 +17,22 @@ Create a new project by running the following command in your terminal:
...
+
+Next you need to add the [Qwik cloudflare-pages adaptor](https://qwik.builder.io/integrations/deployments/cloudflare-pages/#cloudflare-pages-adaptor), you can do so with the following commands:
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-qwik-site.md]:
```suggestion
Next, add the [Qwik cloudflare-pages adaptor](https://qwik.builder.io/integrations/deployments/cloudflare-pages/#cloudflare-pages-adaptor) with the following commands:
```

File: content/pages/framework-guides/deploy-a-qwik-site.md [hunk 4]
```diff
@@ -17,15 +17,22 @@ Create a new project by running the following command in your terminal:
...
+
+```sh
+$ cd qwik-app
+$ npm run qwik add cloudflare-pages
+```
+
+Note that as part of the cloudflare-pages adaptor installation, a `functions/[[path]].ts` file will be created. The `[[path]]` filename indicates that this file will handle requests to all incoming URLs; refer to [Path segments](/pages/platform/functions/#path-segments) to learn more.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-qwik-site.md]:
```suggestion
Note that as part of the `cloudflare-pages` adaptor installation, a `functions/[[path]].ts` file will be created. The `[[path]]` filename indicates that this file will handle requests to all incoming URLs. Refer to [Path segments](/pages/platform/functions/#path-segments) to learn more.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**